tools/SeekRPC/SeekInterface.cpp

```
@@ -79,9 +96,29 @@ bool SeekInterface::isQueryComplete(int64_t task_id) {
         return true;
     }
     return false;
-  }
+}
+
+void SeekInterface::pvalueGenes(const PValueGeneArgs& query, PValueResult& result) {
+    // TaskInfoPtrS task = make_shared<TaskInfo>();
+    // task->queryType = QueryType::Pvalue;
+    // task->pvalueGeneQuery = query;
+    if (this->pvalueEnabled == true) {
+        pvalueGenesCommon(query, result);
+    } else {
+        result.success = false;
+        result.status = QueryStatus::Error;
+        result.statusMsg = "Pvalue server not initialized properly, check pvalue directory";
+        result.__isset.status = true;
+        result.__isset.statusMsg = true;
+    }
+    return;
+}
 
-int32_t SeekInterface::getRpcVersion()
+void SeekInterface::pvalueDatasets(const PValueDatasetArgs& query, PValueResult& result) {
+    printf("pvalueDatasets query not implemented in RPC server yet\n");
```

Maybe return an error?